packages/postgres-database/src/simcore_postgres_database/utils_folders.py

```diff
@@ -464,7 +478,7 @@ async def folder_create(
     *,
     description: str = "",
     parent: _FolderID | None = None,
-    required_permissions: _FolderPermissions = _requires(  # noqa: B008
+    _required_permissions: _FolderPermissions = _requires(  # noqa: B008
```

Comment:
I do not get the point of this change. It's a field with a default value that you don't need to fill. You can ignore it, like you do with other similar parameters